cmd/image-processing/main_test.go

```diff
@@ -399,4 +403,140 @@ var _ = Describe("Image Processing Resource", func() {
 			})
 		})
 	})
+
+	Context("vulnerability scanning", func() {
+		var directory string
+		BeforeEach(func() {
+			cwd, err := os.Getwd()
+			Expect(err).ToNot(HaveOccurred())
+			directory = path.Clean(path.Join(cwd, "../..", "test/data/images/vuln-image-in-oci"))
+		})
+
+		It("should run vulnerability scanning if it is enabled and output vulnerabilities equal to the limit defined", func() {
+			vulnOptions := &buildapi.VulnerabilityScanOptions{
+				Enabled: true,
+			}
+			withTempRegistry(func(endpoint string) {
+				tag, err := name.NewTag(fmt.Sprintf("%s/%s:%s", endpoint, "temp-image", rand.String(5)))
+				Expect(err).ToNot(HaveOccurred())
+				vulnSettings := &resources.VulnerablilityScanParams{VulnerabilityScanOptions: *vulnOptions}
+				withTempFile("vuln-scan-result", func(filename string) {
+					Expect(run(
+						"--insecure",
+						"--image", tag.String(),
+						"--push", directory,
+						"--vuln-settings", vulnSettings.String(),
+						"--result-file-image-vulnerabilities", filename,
+						"--vuln-count-limit", "10",
+					)).ToNot(HaveOccurred())
+					output := filecontent(filename)
+					Expect(strings.Contains(output, "CVE-2019-8457")).To(BeTrue())
```

Comment:
Will produce a better output in case substring is not found:

```suggestion
					Expect(output).To(ContainSubstring("CVE-2019-8457"))
```